src/routes/docs/products/functions/develop-locally/+page.markdoc

```diff
@@ -0,0 +1,114 @@
+---
+layout: article
+title: Develop locally
+description: Learn to develop Appwrite functions locally.
+---
+
+Develop your Appwrite functions locally so you can make code changes without redeploying your function on every code change.
+
+# Setup {% #setup %}
+
+1. Install the [Docker CLI](https://www.docker.com/products/docker-desktop/)
+2. Ensure Docker is running in the background
+3. Install the [Appwrite CLI](/docs/tooling/command-line/installation#installation)
+4. [Log in](/docs/tooling/command-line/installation#get-started) to your Appwrite account using `appwrite login`
+5. [Initialize your project](/docs/tooling/command-line/installation#initialization) and create the `appwrite.json` file
+
+We use Docker to simulate the production environment precisely so that we can deploy a function locally. You can locally run functions using the `appwrite run functions` CLI command. The CLI also allows other [CLI commands](/docs/tooling/command-line/commands), and initializing a project with `appwrite.json` file is required to run commands like `appwrite run functions`.
+
+{% arrow_link href="/docs/tooling/command-line/functions" %}
+Learn more about deploying functions using CLI
+{% /arrow_link %}
+
+# Locally develop your function {% #locally-develop-your-function %}
+
+Use the `appwrite run functions` command to develop your function locally.
+
+```sh
+appwrite run functions
+? Which function would you like to develop locally? My Awesome Function ("<FUNCTION_ID>")
+
+runtime   │ entrypoint  │ path                           │ commands     
+───────────┼──────────────┼───────────────────────────────────┼─────────────
+node-16.0 │ src/main.js │ functions/"<FUNCTION_ID>"        │ npm install
```

Comment:
I don't think the double quotes here are right since the actual output doesn't include double quotes, right?

Reply:
I figured as much, but I wasn't sure because I wanted to maintain consistency with the rest of the docs, but at the same time, I knew that the actual output itself didn't include double quotes, but I double-guessed myself here